.github/workflows/build.yml

```diff
@@ -1,47 +1,56 @@
-name: Build Animate
+name: 'Build Animate'
 
 on:
-  # Push to main or PR
   push:
     branches:
       - main
   pull_request:
 
+concurrency:
+  # Cancel jobs running if new commits are pushed
+  group: >
+    ${{ github.workflow }}-
+    ${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
 jobs:
   build:
-    name: "Build Animate"
-    # The type of runner that the job will run on
+    name: 'Build Animate'
     runs-on: ubuntu-latest
-    # The docker container to use.
     container:
       image: jwallwork/firedrake-parmmg:latest
       options: --user root
     steps:
       - uses: actions/checkout@v2
-      - name: Cleanup
+
+      - name: 'Cleanup'
         if: ${{ always() }}
         run: |
           cd ..
           rm -rf build
-      - name: Setup python
+
+      - name: 'Setup python'
         uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - name: Install Animate
+
+      - name: 'Lint'
+        if: ${{ always() }}
         run: |
           . /home/firedrake/firedrake/bin/activate
-          python -m pip uninstall -y animate
-          python -m pip install -r requirements.txt
-          python -m pip install -e .
-      - name: Test Animate
+          flake8 --ignore=E501,E226,E402,E741,F403,F405,W503
```

Comment:
Are these exceptions not .flake8 already?
It would be annoying if the ones specified here got out of sync with the .flake8
(I know  this was there before)

Reply:
Good point, will use the `.flake8` file you introduced in the CI, too.